### PR TITLE
Feature/wcag label issue

### PR DESCRIPTION
**Pull request by eskilsen**
From FUNKA audit. Some components did not have corresponding id between label and component. 
Task: https://payexjira.atlassian.net/jira/software/c/projects/SWED/boards/312?modal=detail&selectedIssue=SWED-1048
We also understood the error texts are unreachable for screenreaders. This is now fixed by including error message as plain text.
FS170 (https://payexjira.atlassian.net/jira/software/c/projects/SWED/boards/312?modal=detail&selectedIssue=SWED-1041)

## How Has This Been Tested?

Ran all existing tests and update snapshots

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have updated the **CHANGELOG** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: src/App/ComponentsDocumentation/components/Select/index.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ const ContentGuidelines = () => (
         <h2 id="content-guidelines">Content guidelines</h2>
 
         <div className="d-flex justify-content-center slab slab-plain px-5 pt-5 pb-4 no-pointer-events">
```
Comment (PhungNg) [inline src/App/ComponentsDocumentation/components/Select/index.js:60]: I was wondering if we should remove the `no-pointer-events` here 🤔 
Maybe have a quick discussion with Maja about it.

**Review by PhungNg — CHANGES_REQUESTED** [review]
Some changes needed, otherwise, great work! :D

File: src/App/ComponentsDocumentation/components/Select/index.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ const WhenToUse = () => (
                         <div className="px-4 slab slab-plain slab-border-error h-100 mb-2 d-flex flex-column">
                             <span className="h4">Don&#x27;t</span>
                             <div >
-                                <InputGroup type="select" label={label} placeholder={placeholder} selectOptions={selectOptions} />
+                                <InputGroup type="select" id="dont-example" label={label} placeholder={placeholder} selectOptions={selectOptions} />
```
[the remark this answers is not in the document]
Comment (eskilsen) [reply src/App/ComponentsDocumentation/components/Select/index.js:24]: Id prop is added to selectattrs. Is this what you meant?

File: src/App/ComponentsDocumentation/components/Select/constants.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ export const selectOverview = {
     id: "no-tabs",
     elements: [
         {
-            component: <InputGroup type="select" label="Label" placeholder={"Select option"} selectOptions={["Blueberry", "Blackberry", "Berryberry"]} />,
+            component: <InputGroup id="select-showcase-example" type="select" label="Label" placeholder="Select option" selectOptions={["Blueberry", "Blackberry", "Strawberry"]} />,
```
[the remark this answers is not in the document]
Comment (eskilsen) [reply src/App/ComponentsDocumentation/components/Select/constants.js:11]: I love it <3

File: src/App/ComponentsDocumentation/components/Select/index.js (hunk 1 above)
Comment (eskilsen) [reply src/App/ComponentsDocumentation/components/Select/index.js:60]: Removed no-pointer-events and added "related option" as options

Comment (eskilsen) [reply src/App/ComponentsDocumentation/components/Select/index.js:60]: OK, let's wait until Maja can answer :)

File: src/App/components/InputGroup/index.js [hunk 4]
```diff
@@ -103,7 +103,7 @@ const InputGroup = ({
                     {type === "textarea" ?
                         <textarea {...attrs}></textarea>
                         : type === "select" ?
-                            <select {...selectAttrs}>{"\n\t\t"}
+                            <select {...selectAttrs} id={id}>{"\n\t\t"} {/* Placeholder for select is removed when ID is included here?? wtf */}
```
[the remark this answers is not in the document]
Comment (eskilsen) [reply src/App/components/InputGroup/index.js:106]: Yes I know, it "reappears".. Wierd 😅

File: src/App/ComponentsDocumentation/components/Select/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (PhungNg) [reply src/App/ComponentsDocumentation/components/Select/index.js:24]: As I mentioned on slack :)

File: src/App/components/InputGroup/index.js [hunk 5]
```diff
@@ -115,7 +115,7 @@ const InputGroup = ({
                             <input {...attrs} />}
                 </>
             }
-            {helpBlock ? <div className="help-block" data-success={successMessage || null} data-error={errorMessage || null}>{helpBlock}</div> : null}
+            {helpBlock ? <div className="help-block" data-success={successMessage || null} data-error={errorMessage || null}>{"\n"}{helpBlock}{errorMessage}</div> : null}
```
[the remark this answers is not in the document]
Comment (PhungNg) [reply src/App/components/InputGroup/index.js]: Checkboxes still gets the `.help-block` content through css.

File: src/App/ComponentsDocumentation/components/Select/constants.js [hunk 6]
```diff
@@ -8,7 +8,7 @@ export const selectOverview = {
     id: "no-tabs",
     elements: [
         {
-            component: <InputGroup id="select-showcase-example" type="select" label="Label" placeholder="Select option" selectOptions={["Blueberry", "Blackberry", "Strawberry"]} />,
+            component: <InputGroup id="select-showcase-example" type="select" label="Label" placeholder="Select option" selectOptions={["Blueberry", "Blackberry", "Berryberry"]} />,
```
Comment (PhungNg) [inline src/App/ComponentsDocumentation/components/Select/constants.js:11]: ❤️

**Review by PhungNg — COMMENTED** [review]
Nice! Just some more changes required :)

**Review by PhungNg — APPROVED** [review]
Great work! :D 👏

**Pull request merged**